cd-ls.plugin.zsh

```diff
@@ -5,6 +5,6 @@ fi
 function chpwd_cdls() {
   if [[ -o interactive ]]; then
     emulate -L zsh
-    ls
+    eval ${CD_LS_COMMAND:=ls}
```

Comment:
Can we make this `${CD_LS_COMMAND:-ls}`? I'd rather this plugin not pollute with extra environment variable assignments.

Reply:
Thanks for pointing it out, it was a typo. Fixed 😆